.openhands/microagents/repo_eng_agent.md

```
@@ -0,0 +1,55 @@
+
+### Proactive Suggestions and Changes
+The agent will make suggestions and commit changes for:
+- New features implementation
+- Code reviews (providing feedback/suggestions)
+- Review comments/suggestions implementation
+- Branch merge conflict resolution
+- Code refactoring
+- Bug fixes
+- Branch cleanups
+
+### Review Management
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)